### ISSUE TITLE
[Orders] SHIPPO OrderShopApp enums missing

### ISSUE THREAD

**Issue by jfriedr** [goshippo/shippo-python-sdk#84]
Getting an exception when running the example code to retrieve list of orders:

https://github.com/goshippo/shippo-python-sdk/blob/main/docs/sdks/orders/README.md#example-usage

```python
import shippo
from shippo.models import components, operations

s = shippo.Shippo(
    api_key_header="<YOUR_API_KEY_HERE>",
    shippo_api_version='2018-02-08',
)


res = s.orders.list(request=operations.ListOrdersRequest(
    order_status=[
        components.OrderStatusEnum.PAID,
    ],
    shop_app=components.OrderShopAppEnum.SHIPPO,
))

if res is not None:
    # handle response
    pass
```

```shell
python get_orders.py 
Traceback (most recent call last):
  File "/Users/<username>/hippo/shippo-python-sdk/.idea/get_orders.py", line 21, in <module>
    shop_app=components.OrderShopAppEnum.SHIPPO,
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

The SHIPPO enum is missing. 

![image](https://github.com/user-attachments/assets/4b268ba3-5cb6-492f-aa4a-03967fb9cbef)

List of enums:

https://github.com/goshippo/shippo-python-sdk/blob/main/docs/models/components/ordershopappenum.md

**Comment by jfriedr** [comment]
![image](https://github.com/user-attachments/assets/eddb7d35-4ed1-432f-8688-b8a32d8ed635)

**Comment by DLTADragonHawk** [comment]
@jfriedr I was able to run the example code as is but, only after an update from 3.5.2->3.7.0.  It looks like the OrderShopAppEnum component didn't exist in the previous version.  Could you confirm which version you are trying this on?

**Comment by jfriedr** [comment]
🤔 Thanks for pointing that out, it looks like I was pointed to an older spec when trying to replicate another issue. The version was `3.5.1`.

Thanks 🙏
